test/matpower.jl

```diff
@@ -105,7 +105,7 @@ end
     ExaPF.get!(polar, State(), x, cache)
 
     @test conv.n_iterations == 5
-    @test isapprox(norm(cache.balance, Inf), 1.3783e-12, rtol=1e-2)
+    @test isapprox(norm(cache.balance, Inf), 1.3783e-12, rtol=1e-1)
```

Comment:
Mmmmh. thinking more about it, the residual should be close to 0 at the end. At the moment we are comparing with the final value we get with matpower, but I think it would make more sense to compare it with 0: 
`@test isapprox(norm(cache.balance, Inf), 0, rtol=1e-1)`
Not sure that `rtol` makes any sense for numerical values this small :-)

Reply:
@michel2323 @frapac this makes sense (comparing with 0).

Its however intriguing, that the final residual value seemed to be quite stable/robust along different platforms (my laptop, desktop, moonshot...).

Reply:
Fixed according to @frapac 's suggestions. @dmaldona Summit uses PPC CPUs. So this is the first system with a different architecture and instruction set.